scripts/update_and_rebuild_libmesh.sh

```diff
@@ -66,7 +66,22 @@ fi
 # generate machine diagnostics and write it to the log
 $SCRIPT_DIR/diagnostics.sh > "$SCRIPT_DIR/$DIAGNOSTIC_LOG"
 
+# Checks whether or not LIBMESH_DIR is a child of the conda base
+function check_libmesh_dir_in_conda()
```

Comment:
This is a function because it makes it much shorter being able to do `|| return` instead of `if [ $? == 0]` everywhere.